common/components/ui/Code.scss

```diff
@@ -1,14 +1,42 @@
-pre {
+@import 'common/sass/variables';
+@import 'common/sass/mixins';
+
+code {
```

Comment:
Since this is componentized, we should target classnames here. I wouldn't want someone relying on using the raw `<code/>` and `<pre/>` tags without going through the component.